dataset-spec/README.md

```diff
@@ -0,0 +1,92 @@
+
+## Core
+
+| Element       | Type                                  | Name                            | Description                                                  |
+| ------------- | ------------------------------------- | ------------------------------- | ------------------------------------------------------------ |
+| id            | string                                | Dataset ID (required)           | Identifier for the dataset that is unique across the provider. MUST follow the pattern ` ^[A-Za-z0-9_\-\/]+$ `. TODO: Allow slash? |
+| title         | string                                | Title                           | A short descriptive one-line title for the dataset.          |
+| description   | string                                | Description (required)          | Detailed multi-line description to fully explain the entity. [CommonMark 0.28](http://commonmark.org/) syntax MAY be used for rich text representation. |
+| keywords      | [string]                              | Keywords                        | List of keywords describing the dataset.                     |
+| version       | string                                | Dataset Version                 | Version of the dataset. [Semantic Versioning (SemVer)](https://semver.org/) SHOULD be followed. |
+| license_name  | string                                | Dataset License Name (required) | Dataset's license name based on [SPDX License Identifier](https://spdx.org/licenses/) or `proprietary` (see `license_url`). TODO: How to handle non SPDX? internal, proprietary, ...? |
+| license_url   | string                                | Dataset License URL             | Dataset's license URL SHOULD be specified if `license_name`  does not contain a SPDX License Identifier. |
+| provider      | [Provider Object]                     | Data Provider                   | The organization that creates the content of the dataset.    |
+| host          | Host Object                           | Storage Provider                | The organization that hosts the dataset.                     |
+| geometry      | [GeoJSON Object](http://geojson.org/) | Spatial extent (required)       | The spatial extent covered by the dataset as [GeoJSON](http://geojson.org/) object. |
+| datetime      | string                                | Temporal extent (required)      | Temporal extent covered by the dataset. Date/time intervals MUST be formatted according to ISO 8601. Open date ranges are not supported by ISO 8601 and MUST be encoded as proposed by [Dublin Core Collection Description: Open Date Range Format](http://www.ukoln.ac.uk/metadata/dcmi/date-dccd-odrf/2005-08-13/). |
+| process_graph | Process Graph Object                  | Processing chain                | ...                                                          |
```

Comment:
What is this? And what's the definition of it? May make sense as an extension?

Reply:
Process graph was added for provenance, but will definitely be an extension. The same applies for dimensions. Will change that, too.